internals/scripts/setup.js

```diff
@@ -68,6 +52,33 @@ function deleteFileInCurrentDir(file, callback) {
 /**
  * Installs dependencies
  */
-function installDeps(callback) {
-  exec('npm install', addCheckMark.bind(null, callback));
+function installDeps() {
+  exec('yarn --version', function (err, stdout, stderr) {
+    if (parseFloat(stdout) < 0.15 || err || process.env.USE_YARN === 'false') {
```

Comment:
Morning @mxstbr 
Yarn from now on by default will be installed as 0.15 and above and all the tests that i have done were done with this version. With previous versions there might be unexpected behaviors. Do you think a comment like this will do?
`we expect yarn ^0.15 because previous versions were not tested and released before open sourcing yarn`